prepare_release.csx

```diff
@@ -146,10 +146,8 @@ public static async Task UpdateIssue(Issue existingIssue, Milestone existingMile
 public static async Task CreateNextIssue(Issue existingIssue, Milestone nextMilestone)
 {
     var newIssue = new NewIssue("Release vNext") { Milestone = nextMilestone.Number, Body = existingIssue.Body.Replace("[x]", "[ ]") };
-    foreach (var label in existingIssue.Labels)
-    {
-        newIssue.Labels.Add(label.Name);
-    }
+    newIssue.Labels.Add("build");
+    newIssue.Labels.Add("documentation");

```

Comment:
Minor nitpick: for consistency, you could use the object initializer syntax for labels too:

    var newIssue = new NewIssue("Release vNext")
    {
        Milestone = nextMilestone.Number,
        Body = existingIssue.Body.Replace("[x]", "[ ]"),
        Labels = { "build", "documentation" }
    };

Reply:
I avoided that because `Labels` appears to be readonly:

https://github.com/octokit/octokit.net/blob/master/Octokit/Models/Request/NewIssue.cs#L67

```c#
public Collection<string> Labels { get; private set; }
```

But I tried it, and... it worked! I'm a little confused. (also sorry about mucking the main repo - I'd accidentally reverted my "repoOwner" setting).

Reply:
> But I tried it, and... it worked! I'm a little confused.

Because it doesn't attempt to set the property, it just adds to the existing collection. `Labels = new Collection<string> { "build", "documentation" }` wouldn't have worked, though ;)

Reply:
**What sorcery is this!?!?!**

I did not know this syntax existed. Amending.